linear/link-pr-to-linear-issue/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "link-pr-to-linear-issue",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/index.ts",
+  "scripts": {
+    "test": "vitest run --config vitest.config.ts",
+    "prebuild": "pnpm test",
```

Comment:
Because this repository doesn't have CI workflows for the different actions, I have added a `prebuild` step to indicate to maintainers that tests actually exist and that they should pass before `build` outputs are generated.